src/core/Form/DateInput/DateInput.tsx

```diff
@@ -245,6 +248,9 @@ const BaseDateInput = (props: DateInputProps) => {
     datePickerDefaultTexts.fi,
   );
 
+  // Remove the possibility to have undefined forwardedRef as a parameter for forkRefs
+  const definedRef = forwardedRef || null;
```

Comment:
Maybe forwardedRef could be set in props to default null to handle this also?

Reply:
Probably would work, yes. I just copied this line from other components where `forkRefs` is used

Reply:
Ok it seems a bit verbose but it does make it more clear why type is being changes here. 👌